db/data/regions/CH/en.yml

```diff
@@ -8,10 +8,12 @@ en:
             default: Street and house number
             optional: Street and house number (optional)
           errors:
-            blank: Enter a street and house number
+            blank_instructional: Enter a street and house number
+            blank_informative: Street and house number is missing
         address2:
           label:
             default: Additional address
             optional: Additional address (optional)
           errors:
-            blank: Enter additional address information
+            blank_instructional: Enter additional address information
+            blank_informative: Additional address information is missing
```

Comment:
Maybe this is more a comment for @ksvihar, but I'm wondering if this is a meaningful or necessary message to communicate (in either instructional or informative variants) if the field is optional.